### PR TITLE
Fix test failures with Stylelint 16.7.0

### PR DESCRIPTION
**Pull request by ybiquitous**
Ref https://github.com/stylelint/stylelint/releases/tag/16.7.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
 		"editorconfig-checker": "^5.1.2",
 		"eslint": "^8.56.0",
 		"husky": "^8.0.3",
-		"stylelint": "^16.0.2"
-	}
+		"stylelint": "^16.7.0"
+	},
+	"packageManager": "pnpm@9.5.0+sha512.140036830124618d624a2187b50d04289d5a087f326c9edfc0ccd733d76c4f52c3a313d4fc148794a2a9d81553016004e6742e8cf850670268a7387fc220c903"
```
Comment (ybiquitous) [inline package.json]: [ask] Can you tell me how to prevent `packageManager` is automatically inserted by the `pnpm i` command?

**Review by firefoxic — APPROVED** [review]
@ybiquitous Hi! Thanks for PR!

File: package.json (hunk 1 above)
Comment (firefoxic) [reply package.json]: To do this, you just need to specify the required version without a checksum:
```suggestion
	"packageManager": "pnpm@9.6.0"
```

**Comment by ybiquitous** [comment]
@firefoxic Thank you for the quick merge! 😄

**Pull request merged**